spec/index.bs

```diff
@@ -134,7 +134,16 @@ spec: Strings on the Web; urlPrefix: https://w3c.github.io/string-meta/#
         text: best practices for language and direction information; url: bp_and-reco
 </pre>
 
+<link rel="icon" type="image/png" sizes="32x32" href="img/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="96x96" href="img/favicon-96x96.png">
+
 <style>
+/* Displays the WebXR logo next to the W3C logo */
```

Comment:
```suggestion
/* Displays the WebGPU logo next to the W3C logo */
```

Reply:
Derp. I _did_ say it was based on that one! 😆